v2/anomalydetection/sidebars.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  sidebar: [
+    "introduction",
+    {
+      type: "category",
+      label: "Setup Guide",
+      collapsed: false,
+      items: ["frontend-setup", "backend-setup"],
+    },
```

Comment:
dont think you need to have this nesting.

Reply:
done